aep/general/0134/aep.md.j2

```diff
@@ -194,14 +199,7 @@ will create the resource in the process):
 
 ```proto
 message UpdateBookRequest {
-  // The book to update.
-  //
-  // The book's `path` field is used to identify the book to be updated.
-  // Format: publishers/{publisher}/books/{book}
-  Book book = 1 [(google.api.field_behavior) = REQUIRED];
-
-  // The list of fields to be updated.
-  google.protobuf.FieldMask update_mask = 2;
+  ...
```

Comment:
nit: have we annotated proto examples to show that they're partial? All of this is fine, but just want to make sure we annotate partial messages consistently

Reply:
no I don't think so - follow-up issue? 

One could argue that the existence of an update_mask field is the proof that partial update is supported..

Reply:
follow-up issue: https://github.com/aep-dev/aep.dev/issues/214

Reply:
Should've been more specific on that. We should ignore this comment. 

I just meant that you used the '...' notation to show that there's a bunch of implied fields in the message, but they're removed from the example. I was originally thinking we should make sure that notation is clear, but I now think it's plenty clear. Please ignore.